pkg/apis/keycloak/v1alpha1/keycloakbackup_types.go

```diff
@@ -7,16 +7,53 @@ import (
 // KeycloakBackupSpec defines the desired state of KeycloakBackup
 // +k8s:openapi-gen=true
 type KeycloakBackupSpec struct {
-	Restore bool            `json:"restore,omitempty"`
-	AWS     KeycloakAWSSpec `json:"aws,omitempty"`
+	// Controls automatic restore behavior.
+	// Currently not implemented (yet)
```

Comment:
Remove (yet). Perhaps add a short paragraph on how it is envisioned to work in the future.

Reply:
Fixed.